### PR TITLE
fix(curriculum): replace js with full solution in accessibility challenge

### PR DESCRIPTION
**Pull request by effendiian**
This commit replaces a JavaScript only solution where a full, working
solution should have been written.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to issue #35751

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast.english.md [hunk 1]
```diff
@@ -69,7 +69,24 @@ tests:
 
 
 ```js
```
Comment (thecodingaviator) [inline curriculum/challenges/english/01-responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast.english.md]:
```suggestion
```html
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by rimij405** [comment]
Hi @thecodingaviator - changes have been applied and are waiting approval.

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Comment by ojeytonwilliams** [comment]
@rimij405  Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:

**Pull request merged**